### PR TITLE
Fix video in new window issue

### PR DESCRIPTION
**Pull request by PikachuEXE**
**Important note**
Please note that only PrestoN is able to merge Pull Requests into master.

**Pull Request Type**
Please select what type of pull request this is:
- [x] Bugfix

**Related issue**
#1636

**Description**
Fix issue after opening a new window via middle mouse button click on a video

**Screenshots (if appropriate)**
https://user-images.githubusercontent.com/1018543/132179632-77f31596-b37b-4c79-80e6-37d4e82e128c.mov

**Testing (for code that is not small enough to be easily understandable)**
- Go Trending
- Click middle mouse button on any video to open a new video

**Desktop (please complete the following information):**
 - OS: MacOS
 - OS Version: 11.5.2
 - FreeTube version: based on 818fa687f350969e3862ba00a4b052b15418611b

**Additional context**

### CODE REVIEW

**Comment by peepopoggers** [comment]
This is for @PrestonN to decide

File: package.json [hunk 1]
```diff
@@ -107,7 +107,7 @@
     "babel-loader": "^8.2.2",
     "copy-webpack-plugin": "^9.0.1",
     "css-loader": "5.2.6",
-    "electron": "^14.0.0",
+    "electron": "^13.1.7",
```
Comment (ChunkyProgrammer) [inline package.json:110]: Why not 13.3.0 ? (is the issue present on that version?)

Comment (PikachuEXE) [reply package.json:110]: This change is made by reverting [the diff for the line of electron version update](https://github.com/FreeTubeApp/FreeTube/commit/0db6dd1f970747e34336e964245350cbfdcf7f21#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L110)

I can make it `^13.3.0` but it's almost the same as `^13.1.7`
- `^13.3.0`: `>=13.3.0 <14.0.0`
- `^13.1.7`: `>=13.1.7 <14.0.0`

If you look at https://github.com/FreeTubeApp/FreeTube/pull/1658/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519R6721
`electron`'s version is updated to `13.3.0` anyway (or the latest 13.x if there were newer versions)

Comment (PrestonN) [reply package.json:110]: Let's go ahead and update it to `13.3.0` assuming it doesn't cause the same issues with `14.0.0`. Outside of that, I have no issues with moving forward with this.

**Comment by PikachuEXE** [comment]
Updated to `^13.3.0`

**Comment by PrestonN** [comment]
:+1:

**Pull request merged**